prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -199,7 +199,8 @@ class ActionHistory {
      */
     fun trackActionResponse(response: HttpResponseMessage, verboseResponse: String) {
         action.httpStatus = response.status.value()
-        action.actionResponse = JSONB.valueOf(verboseResponse)
+        if (!verboseResponse.isNullOrBlank())
+            action.actionResponse = JSONB.valueOf(verboseResponse)
```

Comment:
An empty string is not valid json.   this fixes a bug #2178 that occurred in production - there are valid paths thru the code that do not generate a non-empty json action_response field.